From 1870e6c38c0925f4cf7be93241d9e905a2c92795 Mon Sep 17 00:00:00 2001
From: Takeshi Kihara <takeshi.kihara.df@renesas.com>
Date: Fri, 7 Dec 2018 16:25:36 +0900
Subject: [PATCH 664/909] arm64: dts: r8a77990-es10-ebisu: Enable ADSP

commit ebce392436befa3442feb0f657bfeb146ce90379 from
git://git.kernel.org/pub/scm/linux/kernel/git/horms/renesas-bsp.git

This patch enables Audio DSP (ADSP) for the Ebisu board on the R8A77990
ES1.0 SoC.

Signed-off-by: Takeshi Kihara <takeshi.kihara.df@renesas.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/renesas/r8a77990-es10-ebisu.dts | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/arch/arm64/boot/dts/renesas/r8a77990-es10-ebisu.dts b/arch/arm64/boot/dts/renesas/r8a77990-es10-ebisu.dts
index 859e8549a539..087d5533f3db 100644
--- a/arch/arm64/boot/dts/renesas/r8a77990-es10-ebisu.dts
+++ b/arch/arm64/boot/dts/renesas/r8a77990-es10-ebisu.dts
@@ -342,6 +342,11 @@
 	};
 };
 
+&adsp {
+	status = "okay";
+	memory-region = <&adsp_reserved>;
+};
+
 &audio_clk_a {
 	clock-frequency = <22579200>;
 };
-- 
2.17.1

